src/pages/protected/VerordeningEdit/Form/FormArticleSidebar/FormArticleSidebar.tsx

```diff
@@ -0,0 +1,266 @@
+import { FieldLabel, FormikDate, Text } from '@pzh-ui/components'
+import { AngleDown, AngleUp } from '@pzh-ui/icons'
+import classNames from 'classnames'
+import { useFormikContext } from 'formik'
+import { motion } from 'framer-motion'
+import { FC, Fragment, useRef, useState } from 'react'
+
+import { useGetGebruikers } from '@/api/fetchers'
+import { FormFieldWerkingsgebied } from '@/components/Form'
+import FormikSelectUser from '@/components/Form/FormikSelectUser'
+
+import { FormikValues } from '../../verordeningEditContext'
+
+export interface FormArticleSidebarProps {}
+
+function FormArticleSidebar({}: FormArticleSidebarProps) {
+    const { values, setFieldValue } = useFormikContext<FormikValues>()
+    const { data: users } = useGetGebruikers()
+
+    const [allSubSectionsHaveSameGeoArea, setAllSubSectionsHaveSameGeoArea] =
+        useState(false)
+
+    const options = users
+        ? users.map(user => ({
+              value: user.UUID || '',
+              label: user.Gebruikersnaam || '',
+              role: user.Rol || '',
+          }))
+        : []
+
+    if (!values) return null
+
+    return (
+        <SidebarScrollContainer>
+            <SidebarContainer
+                title={`Eigenschappen Artikel ${values.Volgnummer}`}>
+                <div>
+                    <FieldLabel name="Eigenaar_1" label="Eerste Eigenaar" />
+                    <FormikSelectUser
```

Comment:
@Aidenbuis Can all of these FormikSelectUser components be replaced with FormikSelect from @pzh-ui/components?

Reply:
Will be tackled in the implement-gebiedsprogramma's branch